layout/paypal.ejs

```diff
@@ -0,0 +1,39 @@
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="paypal">
```

Comment:
I would like to approve this, and everything else LGTM, but honestly the sandbox functionality is a deal breaker for me. Can't we not pass in a param saying the code is in debug mode?